TSC-1/README.md

```diff
@@ -64,6 +64,18 @@ The typical process is:
 - The TSC discusses and refines specifications during its meetings
 - Accepted specifications are placed on the [roadmap](https://github.com/opendatakit/roadmap/projects/1)
 
+## Requesting feedback
+
+The TSC wants to provide thoughtful asynchronous feedback to committers and other TSC members, but we also don't want to block work.
+
+If feedback from the TSC is needed, it should be in the form of a proposal from a committer or TSC member and it should have a deadline included. The TSC must be notified by being @mentioned in a post on the forum. Everyone on the forum can @mention the TSC and the notifications will also appear in #tsc-1-pulse on Slack.
```

Comment:
Change "If feedback from the TSC is needed, it should be in the form of a proposal from" to "If a decision from the TSC is called for in a given matter, a proposal should be made by a"